nsls2forge_utils/io.py

```diff
@@ -0,0 +1,34 @@
+
+
+def read_file_to_list(path):
+    '''
+    Reads file and places contents in a list line by line.
+
+    Parameters
+    ----------
+    path: str
+        Path to file to read from.
+
+    Returns
+    -------
+    list
+        Contains contents of file line by line.
+    '''
+    with open(path, 'r') as fp:
+        return fp.readlines()
+
+
+def _write_list_to_file(contents, path):
```

Comment:
I wonder if it makes sense to add an optional flag (defaulting to `False`) to sort the `contents` within this function? That way it can be handled in one place.